arcgis-android-toolkit/src/main/java/com/esri/arcgisruntime/toolkit/control/JoystickSeekBar.kt

```diff
@@ -0,0 +1,231 @@
+         *
+         * @since 100.6.0
+         */
+        private const val DEFAULT_MAX = 100.0f
+
+        /**
+         * Default interval value in milliseconds between delta progress updates.
+         *
+         * @since 100.6.0
+         */
+        private const val DEFAULT_DELTA_INTERVAL_MILLIS = 250L
+    }
+
+    /**
+     * Backing minimum property to not conflict with superclass's min value.
+     *
+     * @since 100.6.0
+     */
+    private var _min: Float = DEFAULT_MIN
+
+    /**
+     * Backing maximum property to not conflict with superclass's max value.
+     *
+     * @since 100.6.0
+     */
+    private var _max: Float = DEFAULT_MAX
+
+    /**
+     * Progress value that is offset based upon the [_min] & [_max] values if they are set. Otherwise
+     * the default max value is used.
+     *
+     * @since 100.6.0
+     */
+    private val offsetProgress: Float
+        get() {
+            return if ((_min != DEFAULT_MIN).or(_max != DEFAULT_MAX)) {
+                _min + ((_max - _min) * (progress * 0.01f))
+            } else {
+                max * (progress * 0.01f)
+            }
+        }
+
+    /**
+     * The property representing the delta of the "thumb" control from the center of the JoystickSeekBar.
+     * Can be listened to by using [addDeltaProgressUpdatedListener].
+     *
+     * @since 100.6.0
+     */
+    private var deltaProgress: Float = 0.0f
+
+    /**
+     * A list of [DeltaProgressUpdatedListener] used to listen for delta progress updates.
+     *
+     * @since 100.6.0
+     */
+    private val deltaProgressUpdatedListeners = ArrayList<DeltaProgressUpdatedListener>()
+
+    /**
+     * Timer used to schedule [deltaTimerTask].
+     *
+     * @since 100.6.0
+     */
+    private val deltaTimer: Timer = Timer()
+
+    /**
+     * TimerTask that notifies registered [DeltaProgressUpdatedListener] of the current [deltaProgress].
+     *
+     * @since 100.6.0
+     */
+    private lateinit var deltaTimerTask: TimerTask
+
+    init {
+        // Set the initial progress to half of the max value
+        progress = (max * 0.5).roundToInt()
+        setOnSeekBarChangeListener(this)
+    }
+
+    /**
+     * Constructor used when instantiating this View directly to attach it to another view
+     * programmatically.
+     *
+     * @since 100.6.0
+     */
+    constructor(context: Context) : super(context)
+
+    /**
+     * Constructor used when defining this view in an XML layout.
+     *
+     * @since 100.6.0
+     */
+    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs) {
+        context.theme.obtainStyledAttributes(
+            attrs,
+            R.styleable.JoystickSeekBar,
+            0, 0
+        ).apply {
+            try {
+                _min = getFloat(
+                    R.styleable.JoystickSeekBar_jsb_min,
+                    DEFAULT_MIN
+                )
+                _max = getFloat(
+                    R.styleable.JoystickSeekBar_jsb_max,
```

Comment:
The project doesn't build for me because `R.styleable.JoystickSeekBar`, `R.styleable.JoystickSeekBar_jsb_min` and `R.styleable.JoystickSeekBar_jsb_max` are unresolved

Reply:
Have you tried `./gradlew clean assemble` ? @gunt0001 pulled and never had an issue.

Reply:
> The project doesn't build for me

@LDuncAndroid I take that back. Builds now I've invalidated caches